src/locales/strings/enUS.json

```diff
@@ -803,8 +803,8 @@
   "wc_smartcontract_network_fee": "Network Fee",
   "wc_smartcontract_max_total": "Max Total",
   "wc_smartcontract_confirmed": "Smart Contract Call confirmed",
-  "token_agreement_modal_title": "ETH Needed To Send Tokens",
-  "token_agreement_modal_body": "ETH is required to pay mining fees when sending tokens. The associated Ethereum wallet must contain a sufficient amount of ETH.\n\nIf you do not have ETH, you can acquire ETH within Edge using the Buy or Exchange function.",
+  "token_agreement_modal_title": "%s Needed To Send Tokens",
+  "token_agreement_modal_body": "%s is required to pay the mining fees when sending tokens. The associated %s wallet must contain a sufficient amount of funds.\n\nIf you do not have %s, you can acquire it within Edge using the Buy or Exchange function.",
```

Comment:
I know this is outside the scope but I have a suggestion regarding the text. The second sentence is redundant and can be replaced saying that we don't get the network fees. Suggestion:

%s is required to pay the network mining fees when sending tokens. These fees are paid directly to the network and are not collected by Edge.\n\nIf you do not have %s, you can acquire it using the Buy or Exchange functions.",